doc/source/language/anonymous.rst

```diff
@@ -0,0 +1,142 @@
+.. _anonymous_functions:
+
+Anonymous Functions (A kind of Delegate)
+========================================
+
+.. versionadded:: 0.19.3
+   The anonymous function feature described on
+   this page did not exist prior to kOS 0.19.3.
+
+.. contents:: Contents
+    :local:
+    :depth: 2
+    
+Overview
+--------
+
+There is a certain kind of :ref:`Delegate <delegates>` in which you
+don't need to actually name the function in question, and can just
+assign it right into a variable *as* a delegate to begin with, or
+pass it *as* a delegate to another function call.
+
+This is referred to as an
+`Anonymous Funcion <https://en.wikipedia.org/wiki/Anonymous_function>`__.
```

Comment:
@hvacengi Ooops.  Spelling.  "Funcion".  Since you already may be reviewing I don't want to push late changes now for something this minor.  Feel free to change it if you wish.